e2e/cypress/integration/messaging/message_channel_draw_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Draw plugin : Post message', () => {
+    before(() => {
+        //Login with admin access -> enable Draw plugin
+        cy.apiLogin('sysadmin').visit('/');
+        cy.switchToPluginManagementEnableDisableDraw('Enable', fileName, fileType);
+
+        // # Login, set display preference and go to /
+        cy.apiLogin('user-1').visit('/');
+        cy.apiSaveMessageDisplayPreference();
```

Comment:
Does it need to change display preference or fine to leave as is?

Reply:
Since we are not changing any preferences as you said it is not needed. Will ensure to logout once test is done